grpc-proto/proto/stargate.proto

```diff
@@ -8,7 +8,7 @@ package stargate;
 import "query.proto";
 
 service Stargate { // Maybe this should be call `Query` or `Cql`?
```

Comment:
Is a query both reads and writes or will we also have a "mutation"? I would suggest not using "Cql" to avoid any confusion.

Reply:
No mutation.